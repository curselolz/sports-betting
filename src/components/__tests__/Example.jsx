import React from 'react';
import { shallow } from 'enzyme';
import ExampleContainer from '../Example/ExampleContainer';

describe('Example component', () => {
    it('should render correctly ', () => {
        const component = shallow(<ExampleContainer />);

        expect(component).toMatchSnapshot();
    });
});
