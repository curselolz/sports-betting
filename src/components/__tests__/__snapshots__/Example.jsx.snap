// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example component should render correctly  1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`MyComponent should render correctly  1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
